content/manuals/desktop/features/gordon.md

```diff
@@ -114,26 +105,11 @@ After signing in to your Docker Account, enable the Docker AI feature:
 1. Open the **Settings** view in Docker Desktop.
 2. Navigate to **Features in development**.
 3. Check the **Enable Docker AI** checkbox.
-4. Select **Apply & restart**.
```

Comment:
I believe users should now click on the "Accept and enable" button before selecting Apply & restart.